src/application.js

```diff
@@ -1237,8 +1218,8 @@ class Application {
      * @param {string} schemaId 
      * @returns {Campaign.XtkSchema} the schema, or null if the schema was not found
      */
-     async getSchema(schemaId) {
-        return this._schemaCache.getSchema(schemaId);
+    async getSchema(schemaId) {
+      return this._getSchema(schemaId);
```

Comment:
something strange here: we call _getSchema which defined as bellow with "// Private function: get a schema without using the cache"?